src/app/genomes/genomes-list.pug

```diff
@@ -23,8 +26,11 @@ mat-table([dataSource]="result" class="genome-table")
         mat-option(value="{{genome.order}}") {{genome.order}}
         mat-option(value="{{genome.family}}") {{genome.family}}
         mat-option(value="{{genome.genus}}") {{genome.genus}}
-  ng-container(matColumnDef="Genbank Version")
-    mat-header-cell(*matHeaderCellDef class="genbank-version-header table-txt-size") Genbank Version
+  ng-container(matColumnDef="Version")
+    mat-header-cell(*matHeaderCellDef class="version-header table-txt-size") Version
+    mat-cell(*matCellDef="let genome" class="version-cell" ) {{genome.version}}
+  ng-container(matColumnDef="GenBank Version")
```

Comment:
I don't think we need both GenBank Version and RefSeq Version. GenBank Version need to be removed.